Rubberduck.Core/UI/About/AboutControlViewModel.cs

```diff
@@ -9,6 +9,8 @@
 
 namespace Rubberduck.UI.About
 {
+    using NLog.Targets;
```

Comment:
The convention has been to put the `using` at the top of file, outside of the `namespace`.